src/runner.js

```diff
@@ -4,13 +4,22 @@ const parser = require('./parser/parser.js');
 const results = require('./results.js');
 const DefaultVariableStorage = require('./default-variable-storage.js');
 const nodeTypes = require('./parser/nodes.js').types;
+const MessageFormat = require('messageformat');
 
 class Runner {
-  constructor() {
+  /**
+   * Creates an instance of Runner.
+   * @param {object} [options] Options for the runner
+   * @param {object} [options.formater] Changing default formater used (MessageFormat)
+   * @param {string} [options.language] Options for the language of the formater
+   * @memberof Runner
```

Comment:
I wonder if this decorator comment should be above the Runner class instead (ln8)

Reply:
Jsdoc Always on constructor, not ont the class itself

Reply:
You can attach to the constructor but it will describe only the class w/ no parameters

Reply:
ah ok, no worries :)